src/main/java/org/apache/commons/configuration2/INIConfiguration.java

```diff
@@ -236,12 +236,27 @@ public class INIConfiguration extends BaseHierarchicalConfiguration implements F
      */
     private String commentCharsUsedInInput = COMMENT_CHARS;
 
+    /**
+     * The flag for decision, whether inline comments on the section line are allowed.
+     */
+    private boolean sectionInLineCommentsAllowed;
+
     /**
      * Create a new empty INI Configuration.
      */
     public INIConfiguration() {
     }
 
+    /**
+     * Create a new empty INI Configuration with option to allow inline comments on the section line.
+     *
+     * @param sectionInLineCommentsAllowed when true inline comments on the section line are allowed
+     * @since
```

Comment:
Version for `since` tags should match `pom.xml` minus `-SNAPSHOT`.

Reply:
Added version 2.9.0